### PR TITLE
15 added visualizations for timelines

### PR DESCRIPTION
**Pull request by kamerynB**

### CODE REVIEW

File: client/src/features/timeline/Container.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+const Chart: React.FC = () => {
+  // const facetNameMap: Map<string, string> = new Map([
+  //   ['itemProductionDate', 'Objects Produced'],
+  //   ['itemEncounteredDate', 'Objects Encountered'],
+  //   ['workCreationDate', 'Works Created'],
+  //   ['workPublicationDate', 'Works Published'],
+  // ])
+
+  console.log()
```
Comment (gigamorph) [inline client/src/features/timeline/Container.tsx:92]: Remove this.

File: client/src/features/timeline/Container.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+} from 'recharts'
+
+import theme from '../../styles/theme'
+import StyledEntityPageSection from '../../styles/shared/EntityPageSection'
+
+const data = [
```
Comment (gigamorph) [inline client/src/features/timeline/Container.tsx:16]: What is this data for? It looks bogus.

File: client/src/features/timeline/Container.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  },
+  {
+    name: '1967 C.E.',
+    itemProductionDate: 3490,
+    workPublicationDate: 4300,
+    itemEncounteredDate: 2100,
+  },
+]
+
+const Chart: React.FC = () => {
```
Comment (gigamorph) [inline client/src/features/timeline/Container.tsx:84]: This component doesn't seem to be used anywhere and looks like a duplicate of Graph. Shouldn't this whole file be deleted?

File: client/src/features/timeline/List.tsx [hunk 4]
```diff
@@ -30,9 +31,22 @@ const HoverableRow = styled(Row)`
 
 const List: React.FC<IProps> = ({
   sortedKeys,
+  yearsArray,
   transformedData,
   searchTags,
 }) => {
+  console.log(yearsArray)
```
Comment (gigamorph) [inline client/src/features/timeline/List.tsx:38]: Please remove.

File: client/src/features/timeline/Container.tsx (hunk 1 above)
Comment (kamerynB) [reply client/src/features/timeline/Container.tsx:92]: I removed the file. It was leftover from a rebasing mistake.

File: client/src/features/timeline/List.tsx (hunk 4 above)
Comment (kamerynB) [reply client/src/features/timeline/List.tsx]: Removed all console.logs

File: client/src/features/timeline/Container.tsx (hunk 2 above)
Comment (kamerynB) [reply client/src/features/timeline/Container.tsx]: I removed this whole file.

**Pull request merged**